getting-started/setup-multi-node-basic.md

```diff
@@ -52,17 +52,9 @@ You can now create distributed hypertables using
 use distributed hypertables as a non-superuser, the user role needs to
 exist on all data nodes and have the correct privileges to use the
 data nodes on the access node. Please refer to the next section for
-instruction on how to create roles on all data nodes and grant data
+instructions on how to create roles on all data nodes and grant data
 node privileges.
 
-It should now be possible to create distributed hypertables, e.g.,
```

Comment:
Forgot to remove this in earlier PR. It's the same information as paragraph above.